Week1/exercise/w1/index.js

```diff
@@ -15,6 +31,19 @@ console.log('header: ', header);
 //    set image src to imageInput element value
 
 // ====================================== //
+document.querySelector('#btn-addTodo');
+const btnToDo = document.querySelector('#btn-addTodo');
+console.log(btnToDo);
+
+function newToDo () {
+    const toDo = document.querySelector('#todoList');
+    const toDoInput = document.querySelector('#todoInput');
+    console.log(toDoInput.value);
+    var li = document.createElement('li');
+    li.innerHTML = toDoInput.value;   
+    document.body.appendChild(li);
```

Comment:
You need to add the \<li\> element to the \<ul\> not the body

Reply:
document.querySelector('#btn-addTodo');
const btnToDo = document.querySelector('#btn-addTodo');
console.log(btnToDo);

function newToDo () {
    const toDo = document.querySelector('#todoList');
    const toDoInput = document.querySelector('#todoInput');
    console.log(toDoInput.value);
    var li = document.createElement('li');
    li.innerHTML = toDoInput.value; 
    ul.appendChild(li);
}
btnToDo.addEventListener('click', newToDo);

Reply:
Don't put code in comments, update your pull request by pushing new commits

Reply:
you are doing `ul.appendChild(li);` but the variable `ul` is not defined. First declare that variable, assign the list element as the value of that variable, then you can append a child element to that list

Reply:
```js 
document.querySelector('#btn-addTodo');
const btnToDo = document.querySelector('#btn-addTodo');
console.log(btnToDo);

function newToDo () {
const toDo = document.querySelector('#todoList');
const toDoInput = document.querySelector('#todoInput');
console.log(toDoInput.value);
var li = document.createElement('li');
li.innerHTML = toDoInput.value;
const ul = document.querySelector('ul');
ul.appendChild(li);
}
btnToDo.addEventListener('click', newToDo);
```

Reply:
I thought ul is defined in the HTML file, so I shouldn't declare it again. But now, I understand. Thank you for warning me.

Reply:
Please do NOT put code in comments, update your pull request instead by making new commits

Reply:
I updated my pull request but it sent my new file to other my pull request called 'homework week-1 is done'. Is it okay now?